A32NX/html_ui/Pages/A32NX_Core/A32NX_FWC.js

```diff
@@ -76,7 +85,12 @@ class A32NX_FWC {
         this._updateButtons(_deltaTime);
         this._updateTakeoffMemo(_deltaTime);
         this._updateLandingMemo(_deltaTime);
-        this._updateAltitudeWarning();
+
+        if (this.updateThrottler.canUpdate(_deltaTime) !== -1) {
+            this._autopilotDisconnect(_deltaTime);
```

Comment:
I think the autopilot disconnect can probably be outside of this, 500ms is a long time before the warning appears.

Reply:
Ok